backend/src/main/kotlin/clarity/backend/entity/ClassroomEntity.kt

```diff
@@ -40,15 +43,28 @@ class ClassroomEntity() {
         val db = DataManager.conn()
         try {
             val statement = db!!.createStatement()
-            val newUUID = UUID.randomUUID().toString()
+            // must check if new string already exists in the table
+            var newId = getRandomString(6)
+            val selectStatement = """
+                SELECT private_code FROM Classroom
+            """.trimIndent()
+            val ids = mutableListOf<String>()
+            val resultCheck = statement.executeQuery(selectStatement)
+            while (resultCheck.next()) {
```

Comment:
why is this a while loop? can't there be at most one conflict based on this logic

i think this could become `if(resultCheck.next()) { ... }`

Reply:
a teacher can have multiple classrooms though right?

Reply:
fixed